cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -26,7 +27,8 @@ spec:
         deployment: fabric-gateway-controller
         application: gateway-operator
         component: fabric-gateway-controller
-        version: {{ $version }}
+        version: "{{ $version }}"
+        mode: "{{ .ConfigItems.fabric_gateway_controller_mode }}"
```

Comment:
does empty string work?

Reply:
if not maybe prefix with single character or so like
```
mode: "_{{ .ConfigItems.fabric_gateway_controller_mode }}"
```

Reply:
`fabric_gateway_controller_mode` is `disabled` by default

Reply:
https://github.com/zalando-incubator/kubernetes-on-aws/blob/061825f64cce867a0b414d5e4d07e449ff781ab8/cluster/config-defaults.yaml#L265-L273